src/weekly_report.js

```diff
@@ -0,0 +1,65 @@
+const moment = require('moment')
+const today = moment()
+const todaysDate = moment()
+const minusSevenDays = todaysDate.subtract(7, 'days')
+console.log('-7: ', minusSevenDays.format("dddd, MMMM Do YYYY, h:mm:ss a"), '\n');
+console.log('todaysDate: ', today.format("dddd, MMMM Do YYYY, h:mm:ss a"));
+
+const lengthOfUse = (User, where) =>
+  User.findOne({where})
+    .then(user => {
+    return Math.abs(todaysDate.diff(user.createdAt, 'days'))
```

Comment:
Indentation off; please use single line implicit return when possible.  With `moment`, you will not need `Math.abs` etc.